nm/src/nm.c

```diff
@@ -404,8 +404,11 @@ _j4status_nm_device_update(J4statusPluginContext *context, J4statusNmSection *se
                 if ( context->formats.up_wifi_tokens & TOKEN_FLAG_UP_WIFI_SSID )
                 {
                     GBytes *raw_ssid = nm_access_point_get_ssid(section->ap);
-                    data.ssid = g_variant_new_take_string(g_strndup(g_bytes_get_data(raw_ssid, NULL),
-                                                                    g_bytes_get_size(raw_ssid)));
+                    if (raw_ssid) {
+                        char *active_ssid_str = nm_utils_ssid_to_utf8(g_bytes_get_data (raw_ssid, NULL),
```

Comment:
Nit: `gchar`, and you can just call it `ssid` or just put the function call in `g_variant_new_take_string()`